cmd/retester/retester.go

```diff
@@ -110,21 +152,53 @@ type retestController struct {
 	usesGitHubApp bool
 	backoff       *backoffCache
 
-	enableOnRepos sets.String
-	enableOnOrgs  sets.String
+	config *Config
+}
+
+func (c *Config) GetRetesterPolicy(org, repo string) (*RetesterPolicy, error) {
```

Comment:
```suggestion
func (c *Config) GetRetesterPolicy(org, repo string) (RetesterPolicy, error) {
```

Why do we return a pointer to `RetesterPolicy`?